src/documentation/runners/dataflow.md

```diff
@@ -4,6 +4,116 @@ title: "Cloud Dataflow Runner"
 permalink: /documentation/runners/dataflow/
 redirect_from: /learn/runners/dataflow/
 ---
-# Using the Cloud Dataflow Runner
+# Using the Google Cloud Dataflow Runner
+
+The Google Cloud Dataflow Runner uses the [Cloud Dataflow managed service](https://cloud.google.com/dataflow/service/dataflow-service-desc). When you run your pipeline with the Cloud Dataflow service, the runner uploads your executable code and dependencies to a Google Cloud Storage bucket and creates a Cloud Dataflow job, which executes your pipeline on managed resources in Google Cloud Platform.
+
+The Cloud Dataflow Runner and service are suitable for large scale, continuous jobs, and provide:
+
+* a fully managed service
+* [autoscaling](https://cloud.google.com/dataflow/service/dataflow-service-desc#autoscaling) of the number of workers throughout the lifetime of the job
+* [dynamic work rebalancing](https://cloud.google.com/blog/big-data/2016/05/no-shard-left-behind-dynamic-work-rebalancing-in-google-cloud-dataflow)
+
+The [Beam Capability Matrix]({{ site.baseurl }}/documentation/runners/capability-matrix/) documents the supported capabilities of the Cloud Dataflow Runner.
+
+## Cloud Dataflow Runner prerequisites and setup
+To use the Cloud Dataflow Runner, you must complete the following setup:
+
+1. Select or create a Google Cloud Platform Console project.
+
+2. Enable billing for your project.
+
+3. Enable required Google Cloud APIs: Cloud Dataflow, Compute Engine, Stackdriver Logging, Cloud Storage, and Cloud Storage JSON. You may need to enable additional APIs (such as BigQuery, Cloud Pub/Sub, or Cloud Datastore) if you use them in your pipeline code.
+
+4. Install the Google Cloud SDK.
+
+5. Create a Cloud Storage bucket.
+    * In the Google Cloud Platform Console, go to the Cloud Storage browser.
+    * Click **Create bucket**.
+    * In the **Create bucket** dialog, specify the following attributes:
+      * _Name_: A unique bucket name. Do not include sensitive information in the bucket name, as the bucket namespace is global and publicly visible.
+      * _Storage class_: Multi-Regional
+      * _Location_:  Choose your desired location
+    * Click **Create**.
+
+For more information, see the *Before you begin* section of the [Cloud Dataflow quickstarts](https://cloud.google.com/dataflow/docs/quickstarts).
+
+### Specify your dependency
+
+You must specify your dependency on the Cloud Dataflow Runner.
+
+```java
+<dependency>
+  <groupId>org.apache.beam</groupId>
+  <artifactId>beam-runners-google-cloud-dataflow-java</artifactId>
+  <version>0.3.0-incubating</version>
```

Comment:
can you use jekyll variable here?
(sorry for forgetting that on a previous one. Update too?)